pkg/handlers/system_intake.go

```diff
@@ -87,10 +90,28 @@ func (h SystemIntakeHandler) Handle() http.HandlerFunc {
 			euaID, ok := appcontext.EuaID(r.Context())
 			if !ok {
 				logger.Error("Failed to get EUA ID from context")
-				http.Error(w, "Failed to PUT system intake", http.StatusInternalServerError)
+				http.Error(w, "Failed to PUT system intake", http.StatusUnauthorized)
 				return
 			}
 			intake.EUAUserID = euaID
+			updatedTime := time.Now().UTC()
+			intake.UpdatedAt = &updatedTime
+
+			// If status is submitted, the user is trying to submit the system intake to CEDAR
+			if intake.Status == models.SystemIntakeStatusSUBMITTED {
```

Comment:
I don't know if we've had this discussion yet, but since we're not changing the intake data with `SUBMIT`, we can potentially just do an action endpoint here. I'm not convinced of either direction, just pointing out that it may be easier than treating it as a `PUT`

Reply:
see my comment https://github.com/CMSgov/easi-app/pull/167#discussion_r419776835, but I've been operating off the assumption that it's just another attribute that the client can trigger to be submitted.

Reply:
@mikena-truss we might be updating the intake data because auto-save happens on 1-sec of inactivity by the user and there's a possibility that a user can navigate through the end of the form faster than the 1 second inactivity autosave.